### DISCUSSION TITLE
Is immich/cli can ignore SSL verify error?

### DISCUSSION THREAD

**Discussion by Triple-Z** [immich-app/immich#8183] [Q&A] [answered]
I can't find something like `--skip-ssl-verify` option in `immich/cli`. I served immich via caddy, which certificate is self-signed.

**Comment by bo0tzz** [comment, marked as answer]
The CLI doesn't have an option like this at the moment, but feel free to PR one. That said, I would recommend just getting valid certificates from Let's Encrypt.

**Comment by e-dorigatti** [comment]
Pass Caddy's root certificate via the environment variable `NODE_EXTRA_CA_CERTS=/path/to/caddy/root.crt`.

This also works in docker:

```
docker run -it --rm \
        -v /your/photo/library:/import:ro \
        -v /path/to/caddy/root.crt:/caddy.crt \
        -e NODE_EXTRA_CA_CERTS=/caddy.crt \
        -e IMMICH_INSTANCE_URL=https://immich/api \
        -e IMMICH_API_KEY=abcd \
        ghcr.io/immich-app/immich-cli:latest \
        upload # etc
```
